ci/jenkins/test-vmc.sh

```diff
@@ -467,6 +473,46 @@ function deliver_antrea {
     done
 }
 
+function deliver-flow-visibility {
+    echo "====== Pulling and delivering images in flow-visbility.yml to all the Nodes======"
+    FV_IMAGES=($(cat $GIT_CHECKOUT_DIR/build/yamls/flow-visibility.yml | grep image: | awk '{print $(NF)}' | xargs))
+    IPs=($(kubectl get nodes -o wide --no-headers=true | awk '{print $6}' | xargs))
+    old_IFS=$IFS
+    for image in "${FV_IMAGES[@]}"; do
+        docker pull $image
+        IFS=:
+        read repo tag <<< "$image"
+        IFS=$old_IFS
+        echo "Saving" $repo:$tag
+        docker save -o flow-visibility.tar $repo:$tag
+        for i in "${!IPs[@]}"
+        do
+            copy_image flow-visibility.tar $repo ${IPs[$i]} $tag false
+        done
+    done
+
+    echo "====== Pulling and delivering images in clickhouse-operator-install-bundle.yaml to all the Nodes======"
+    wget https://raw.githubusercontent.com/Altinity/clickhouse-operator/0.18.2/deploy/operator/clickhouse-operator-install-bundle.yaml -O $GIT_CHECKOUT_DIR/build/yamls/clickhouse-operator-install-bundle.yaml
```

Comment:
maybe we could consider having our own version of the manifest checked-in, to avoid this...

Reply:
Agree, checked in our own version clickhouse-operator-install manifest with our internal image names. It also benefits the CI job no need to download the manifest.